src/lib/mongoBackend/dbConstants.h

```diff
@@ -94,6 +94,8 @@
 #define CSUB_EXPR_GEOREL             "georel"
 
 #define CSUB_THROTTLING              "throttling"
+#define CSUB_COUNTER                 "counter"
+#define CSUB_MAXLIMIT                "maxLimit"
```

Comment:
New field in DB should be described in documentation at https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/admin/database_model.md#csubs-collection

Reply:
Added new field in document https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/admin/database_model.md#csubs-collection